packages/cli/src/commands/storybook.js

```diff
@@ -26,9 +26,14 @@ export const builder = (yargs) => {
       type: 'integer',
       default: 7910,
     })
+    .option('build-directory', {
+      describe: 'Directory in web/ to store static files',
+      type: 'string',
+      default: 'storybook-static',
```

Comment:
@peterp this is the current default behavior. My inclination is to set this as something like `public/storybook`, taking advantage of the Public dir to serve static files. Thoughts either way?

Reply:
Nice, I like that, we just have to remember that this would be a breaking change.

Reply:
changed to public/storybook here https://github.com/redwoodjs/redwood/pull/2176